db-models/comments.sql

```diff
@@ -1,9 +1,12 @@
 -- db-models/comments.sql
 
 CREATE TABLE comments (
-    comment_id SERIAL PRIMARY KEY,          -- Unique for each comment
-    parent_comment_id INTEGER,              -- ID of the parent comment for replies
-    author_name TEXT NOT NULL,              -- Author comment's name
-    content TEXT NOT NULL,                  -- Comment text
-    created_at TIMESTAMP DEFAULT NOW()      --Timestamp when comment was created
+    comment_id INTEGER PRIMARY KEY AUTOINCREMENT,                   -- Unique for each comment, auto-incremented
+    parent_comment_id INTEGER,                                      -- ID of the parent comment for replies
+    parent_listing_id INTEGER NOT NULL,                             -- ID of the parent comment listing, required
+    creator_id INTEGER NOT NULL,                                    -- ID of the author, required
+    content TEXT NOT NULL,                                          -- Comment text
+    created_at TEXT DEFAULT CURRENT_TIMESTAMP NOT NULL,             -- Timestamp when comment was created
+    FOREIGN KEY (creator_id) REFERENCES creators(id),
+    FOREIGN KEY (parent_comment_id) REFERENCES comments(comment_id)
```

Comment:
We should also add a foreign key to the listings table with the `parent_listing_id` column.
`FOREIGN KEY (parent_listing_id) REFERENCES listings(listing_id)`

Reply:
Be sure to add a comma on the end of line 11 as well